pelican/plugins/more_categories/more_categories.py

```diff
@@ -3,7 +3,7 @@
 Title: More Categories
 Description: adds support for multiple categories per article and nested
 categories
-Requirements: Pelican 3.8 or higher
+Requirements: Pelican 4.2.0 or higher
```

Comment:
That should say 4.0.0. Sorry. The reason I changed it is because in the README it said 4.0.0. From what I can tell, it's requiring 4.0 functionality, so I decided to change it. Was that wrong?

Reply:
If the goal is to be able to run `pip install pelican-more-categories` and have everything Just Work™, I believe the next release (tentatively planned as version 4.5) would be required, as the namespace plugin functionality was merged to master after v4.2 was released.

Reply:
@thecaligarmo Changing that to 4.0.0 is fine, absolutely!

@justinmayer Yes, but, users with older versions of pelican could still use this plugin the old-fashioned way right? Also, any particular reason why you want to skip from 4.2 to 4.5, or are you planning intermediate releases?

Reply:
@oulenz: Good point. I agree that specifying 4.0.0 makes more sense. Regarding the next release, current master drops Python 2.7 support and will most likely also drop Python 3.5 support. Plus, there are a [decent number of changes in master since 4.2](https://github.com/getpelican/pelican/compare/4.2.0...HEAD). As I mentioned in [another thread](https://github.com/pelican-plugins/webring/issues/2#issuecomment-614599329), there aren't any changes I consider to be truly backwards-incompatible, so 5.0 seemed too big and an unwarranted jump, while 4.3 didn't seem to capture the extent of the changes, so I split the difference and suggested 4.5. Sometimes strict semantic versioning doesn't fit all situations. 🤷‍♂️